src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -35,6 +32,17 @@ public class HttpProtocolTests : HttpClientTest
             await GetAsyncSuccessHelper(statusLine, expectedStatusCode, expectedReason);
         }
 
+        [Theory]
+        [InlineData("HTTP/1.1 200      ", 200, "     ")]
+        [InlineData("HTTP/1.1 200      Something", 200, "     Something")]
+        public async Task GetAsync_ExpectedStatusCodeAndReason_SuccessSkipForWinHttpHandlerCurlHandler(string statusLine, int expectedStatusCode, string expectedReason)
+        {
+            if (PlatformDetection.IsUap || UseManagedHandler)
```

Comment:
It looks like this is still failing in the latest test runs. Are you sure this check is correct?

Reply:
It's failing on WinRT (UWP) as well. Because of the tooling issue #26361, I'm not able to verify on my local machine. I will disable test on UAP run.